translations/nl.yaml

```diff
@@ -134,6 +134,7 @@ component:
           te komen.
         button: Lees meer over de vereniging
         studiegids: Bekijken op Studiegids
+        studentenWegwijzer: Bekijk op StudentenWegwijzer
```

Comment:
intentionally different from "Bekijken op Studiegids" in the line above, Bekijk vs Bekijken?

Reply:
Good catch. Changed the other one to bekijk. Bit more compact and sounds better to me too.